third_party/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+if(POLICY CMP0091)
+  cmake_policy(SET CMP0091 NEW)
+endif()
+
+project(thirdparty CXX)
+set(CMAKE_MSVC_RUNTIME_LIBRARY "MultiThreaded")
```

Comment:
"MultiThreaded$<$<CONFIG:Debug>:Debug>" may be better.

Reply:
Done